src/video_core/vulkan_common/vulkan_device.h

```diff
@@ -45,6 +45,9 @@ class Device {
     /// Reports a shader to Nsight Aftermath.
     void SaveShader(const std::vector<u32>& spirv) const;
 
+    /// Returns the name of the VkDriverId reported from Vulkan.
+    std::string_view GetDriverIdName() const;
```

Comment:
```suggestion
    std::string_view GetDriverIDName() const;
```

Given we have other functions in the interface with it uppercased